test/fuzz.js

```diff
@@ -20,52 +21,45 @@ const ENCODINGS_0 = [ // from https://github.com/nodejs/node/blob/master/lib/buf
   'base64',
   'base64url',
   'hex'
-]
-const ENCODINGS = []
-for (const encoding of ENCODINGS_0) {
+].filter((encoding) => {
   try {
     Buffer.from('hello', encoding)
-    ENCODINGS.push(encoding)
+    return true
   } catch (_) {
+    return false
   }
-}
+})
+
+function test (buff, encoding = 'utf8') {
+  const str = buff.toString(encoding)
 
-const test = function (entry, encoding = 'utf8') {
   const context = waf.createContext()
-  const r1 = context.run({
-    atk: entry
-  }, TIMEOUT)
-  assert(r1)
-  assert(r1.data || r1.timeout)
-  // FIXME: there is a reporting issue with alternative encodings
-  // const actual = Buffer.from(JSON.parse(r1.data)[0].rule_matches[0].parameters[0].value, encoding);
-  // const expected = Buffer.from(entry, encoding);
-  // assert.strictEqual(Buffer.compare(actual, expected), 0)
-  const r2 = context.run({
-    [entry]: 'value'
-  }, TIMEOUT)
-  assert(r2)
+
+  const r1 = context.run({ value_attack: str }, TIMEOUT)
```

Comment:
You have deleted a `FIXME`, Is it ok?

Reply:
Yes I've talked with Vlad (he wrote that comment) and it's not really relevant anymore.